app/src/androidTest/java/net/skyscanner/backpack/calendar/BpkCalendarTest.kt

```diff
@@ -323,6 +323,19 @@ class BpkCalendarTest : BpkSnapshotTest() {
     snap(wrapWithBackground(calendar))
   }
 
+  @Test
+  @FlakyTest
```

Comment:
Why is this considered flaky?

Reply:
All other tests for this widget are also marked flaky. It seems like the problem is related to the widget itself, not the tests.